src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -2836,8 +2836,8 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
     /** Concatenate strings separated by spaces */
     private def compose(ss: String*) = ss filter (_ != "") mkString " "
 
-    def isSingletonExistential =
-      nme.isSingletonName(name) && (info.upperBound.typeSymbol isSubClass SingletonClass)
+    def isSingletonExistential: Boolean =
+      nme.isSingletonName(name)
```

Comment:
Wow, really surprised this doesn't change any tests.

Reply:
I guess it didn't make any sense